packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -239,4 +241,10 @@ const onShowSettingsDialog = (target: AvailableSettingsMenuKeys) => {
   showSettingsDialog.value = true
   settingsDialogTarget.value = target
 }
+
+onResult(() => {
```

Comment:
IIRC you can get the result from the arguments of this callback, no need to use initialQueryResult